### PR TITLE
Update marathon readme to reflect https support

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

File: collectd-marathon/20-collectd-marathon.conf [hunk 1]
```diff
@@ -7,7 +7,9 @@
   Import "marathon"
   LogTraces true
   <Module "marathon">
-    host  ["localhost", "8080", "username", "password"]
+    # The last config is optional. It is "false" by default. Set it to "true" when
+    # operating DC/OS in strict mode
+    host  ["http","localhost", "8080", "username", "password","false"]
```
Comment (charliesignalfx) [inline collectd-marathon/20-collectd-marathon.conf]: minor nit, but there should be a space between the comma and the quote before localhost and a space between the comma and quote before false

File: collectd-marathon/README.md [hunk 2]
```diff
@@ -69,7 +69,9 @@ Using the sample configuration file [20-collectd-marathon.conf](https://github.c
 | Import | Path to the name of the pythom module with out the .py extension | `marathon` |
 | LogTraces | Logs traces from the plugin's execution | `true` |
 | verbose | Turns on verbose log statements | `False` |
-| host | A python list of `["<host>", "<port>", "username", "password"]`.  The `username` and `password` are only required for Basic Authentication with the Marathon API. |  no default |
+| host | A python list of `["<scheme>", "<host>", "<port>", "username", "password", "<is_dcos_strict_mode>"]`. `scheme` is either "http" or "https". The `username` and `password` are only required for Basic Authentication with the Marathon API. `is_dcos_strict_mode` is a string that takes "True" or "False". Set `is_dcos_strict_mode` to "True" and `scheme` to "https" if operating DC/OS in strict mode |  no default |
```
Comment (charliesignalfx) [inline collectd-marathon/README.md]: Let's add a column with example values for these ^.

File: collectd-marathon/README.md [hunk 3]
```diff
@@ -69,7 +69,9 @@ Using the sample configuration file [20-collectd-marathon.conf](https://github.c
 | Import | Path to the name of the pythom module with out the .py extension | `marathon` |
```
Comment (charliesignalfx) [inline collectd-marathon/README.md]: This is a mistake I made.  Could you please correct the word "pythom" to "python".

**Comment by molner** [comment]
LGTM - will wait for @charliesignalfx to sign off on the changes he requested prior to merging.

**Comment by molner** [comment]
His request has been satisfied. I'll merge

**Pull request merged**